server/index.js

```diff
@@ -155,6 +159,11 @@ module.exports = (_config = {}) => {
       )}. Will set to 'default' instead.`
     );
     clientConfig = clientConfigs.find(cfg => cfg.str('id') === 'default');
+    if (!clientConfig) {
+      logger.warn('Could not find default client config.');
+      clientConfig = clientConfigs[0];
```

Comment:
It seems like `clientConfig` could end up as `undefined` here on an edge case and then `undefined.str()` would break

Reply:
This actually was happening before in the case where there are no client configs. There is a check now in loadConfigs where it will exit if there is no client config, but I can add an assert here as well to make sure that it only gets this far if `clientConfigs.length`. 

FWIW, this should hopefully only be temporary as eventually we'll have support for the first config to be added via the front end but I figured it was worth adding these checks for now to avoid onboarding friction.